### PR TITLE
Minimal Setup of frontpage

### PR DESCRIPTION
**Pull request by gdegas**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,19 +1,67 @@
...
+    "ecmaFeatures": {
+      "jsx": true
+    }
+  },
   "env": {
+    "es6": true,
     "browser": true,
     "node": true,
-    "es6": true
+    "jquery": true,
```
Comment (jorge-autogravity) [inline .eslintrc:21]: why are you using jquery?

File: .eslintrc [hunk 2]
```diff
@@ -1,19 +1,67 @@
...
+    "mocha": true
```
Comment (jorge-autogravity) [inline .eslintrc:22]: have you tried jest?

File: .eslintrc (hunk 1 above)
Comment (gdegas) [reply .eslintrc:21]: Just for the Bootstrap JavaScript plugin

File: .eslintrc (hunk 2 above)
Comment (gdegas) [reply .eslintrc:22]: I have not tried it yet, but I will take a look!

File: .eslintrc (hunk 1 above)
Comment (jorge-autogravity) [reply .eslintrc:21]: why not use react-bootstrap?

Comment (jorge-autogravity) [reply .eslintrc:21]: this will potentially mess up your react shadow dom diffing algorithm

**Pull request merged**